spotify-app-remote/build.gradle.kts

```diff
@@ -0,0 +1,2 @@
+configurations.maybeCreate("default")
```

Comment:
where did you learn about this from?

Reply:
https://stackoverflow.com/questions/60878599/error-building-android-library-direct-local-aar-file-dependencies-are-not-supp

I saw it in the accepted answer here